### DISCUSSION TITLE
Inquiry about the artifact build history

### DISCUSSION THREAD

**Discussion by hyeyoung-leee** [goharbor/harbor#20717] [Q&A] [answered]
Hi,

Is `build_history` not provided if the artifact's media type is `application/vnd.docker.distribution.manifest.list.v2+json`?

**Comment by chlins** [comment, marked as answer]
The media type `application/vnd.docker.distribution.manifest.list.v2+json ` shows that this is an index manifest, it is just a JSON file records the manifest corresponding to different platform architectures under the image, no build history in it, build_history need to view the specific platform image.
